sefaria/model/text.py

```diff
@@ -6058,6 +6098,20 @@ def get_multi_title_regex_string(self, titles, lang, for_js=False, anchored=Fals
 
     # do we want to move this to the schema node? We'd still have to pass the title...
     def get_regex_string(self, title, lang, for_js=False, anchored=False, capture_title=False, parentheses=False):
+        """
+        Given a string with a Ref, it returns a regex for the Ref.
```

Comment:
This function takes the title of a book as input, not a ref (although titles can be refs)

Reply:
Ah good point! Thanks Noah!